app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :author_id
+  belongs_to :post, foreign_key: :post_id
+
```

Comment:
- [Optional] Consider adding an after_save method to update comments counter after the comment has been saved. Also consider making the update counter method private since you don't need to access it outside this class.